CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ Added:
 Fixed:
 * Remove invalid fields from `chronosphere_dataset`, `chronosphere_trace_tail_sampling_rules`, `chronosphere_trace_metrics_rule` resources.
 
+* Update link to documentation in mapping rules example file.
+
```

Comment:
not a user-facing change (the examples are currently internal only) so no need for this.
```suggestion
```

Reply:
Thanks @prashantv -- I reverted the commit that updated the CHANGELOG. 

Question: Is updating the CHANGELOG a _requirement_? There's a CI check for updating that file, which is the only reason I made the change.

Reply:
We generally require a CHANGELOG update, unless a "skip-changelog" tag is applied. The goal is that users making a change think about whether a changelog update is needed since it's easy to forget, but allow them to opt-out.